python/tests/test_approximate_nearest_neighbors.py

```diff
@@ -436,6 +486,76 @@ def assert_row_equal(r1: Row, r2: Row) -> None:
         assert knn_model._cuml_params["metric"] == metric
 
 
+@pytest.mark.parametrize(
+    "combo",
+    [
+        (
+            "ivfflat",
+            "array",
+            10000,
+            None,
+            "euclidean",
+        ),
+        (
+            "ivfflat",
+            "vector",
+            2000,
+            {"nlist": 10, "nprobe": 2},
+            "euclidean",
+        ),
+        (
+            "ivfflat",
+            "multi_cols",
+            5000,
+            {"nlist": 20, "nprobe": 4},
+            "euclidean",
+        ),
+        (
+            "ivfflat",
+            "array",
+            2000,
+            {"nlist": 10, "nprobe": 2},
+            "sqeuclidean",
+        ),
+        ("ivfflat", "vector", 5000, {"nlist": 20, "nprobe": 4}, "l2", 0.95, 1e-4),
```

Comment:
This has two additional components compared to the others.  Intended?

Reply:
deleted. typo